docs/document/docs/components/desktop/user-org-group-select.md

```diff
@@ -0,0 +1,105 @@
+| id | string | ""  | Component id name | |
+| label | string | ""  | Label for the component | Label is not displayed if unspecified or empty |
+| placeholder | string | "" | Placeholder text displayed in the input field | |
+| disabled | boolean | false | Enable/Disable the component | |
+| requiredIcon | boolean | false | Show/Hide the required icon | |
+| visible | boolean | true | Show/Hide the component | |
+| items | Array\<[Item](#item)\> | []  | List of options to display | Will result an error if the value of items is not an array |
+| value |  Array\<string\> | [] | Selected value | No option will be selected if the `value` are unspecified<br/>Will result an error if the value is not an array |
+
+### Item
+
+| Name   | Type | Default | Description | Remark |
+| :--- | :--- | :--- | :--- | :--- |
+| label | string | null  | Text label for each option | If `Item.label` is unspecified, the value of `Item.value` is displayed on the UI |
+| type | string | null | Icon type of each item | The Item.type property determines the small icon type displayed for each option in both the dropdown menu and the selected items list.<br/>Displays different icons based on different values of the icon property.<br/>Available options:<li>"user" : ![user](/img/selected-user.png)</li><li>"org" : ![org](/img/selected-org.png)</li><li>"group" : ![group](/img/selected-group.png)</li><li>"" : No icon</li> |
+| value | string | null  | Value of each option | Will result an error if setting duplicated value in `Item.value` |
+| disabled | boolean | false | Enable/Disable the item | |
+
+### Event
+
+Here is a list of events that can be specified:
+
+| Name | Type | Description | Remark |
+| :--- | :--- | :--- | :--- |
+| change | function | Event handler when the value has been changed | It will pass the event object as the argument<br/><br/>You can receive the following values when used in event.detail<br/>event.detail.oldValue : Value before the change<br/>event.detail.value : Value after the change |
+| click-picker-icon | function | Event handler when the picker icon is clicked | It will pass the event object as the argument |
+
+### Constructor
+
+UserOrgGroupSelect(options)<br/>
+Here is a list of available constructors:
+
+#### Parameter
+| Name | Type | Default | Description | Remark |
+| :--- | :--- | :--- | :--- | :--- |
+| options | object | \{\} | Object that includes component properties | |
+
+## Sample Code
+
+:::tip
+Please check the [package installation](../../getting-started/quick-start.md#installation) method first.
+:::
+
+Here is a sample code when all parameters are specified:
+
+```javascript
+const Kuc = Kucs['1.x.x'];
+
+const space = kintone.app.record.getSpaceElement('space');
+
+const userSelect = new Kuc.UserOrgGroupSelect({
+  label: 'User Select',
+  items: [
```

Comment:
Please use the reasonable user, org, group name for label and value as same as component preview.
e.g. Sales department, John Brown, etc